cppcache/integration-test/fw_dunit.cpp

```diff
@@ -688,36 +680,63 @@ void sleep(int millis) {
 }
 
 void logCoordinator(std::string s, int lineno, const char * /*filename*/) {
-  char buf[128] = {0};
-  dunit::getTimeStr(buf, sizeof(buf));
-
-  fprintf(stdout, "[TEST coordinator:pid(%d)] %s at line: %d\n",
-          boost::this_process::get_id(), s.c_str(), lineno);
-  fflush(stdout);
+  using std::chrono::duration_cast;
+  using std::chrono::microseconds;
+  using std::chrono::system_clock;
+
+  auto now = system_clock::now();
+  auto in_time_t = system_clock::to_time_t(now);
+  auto localtime = std::localtime(&in_time_t);
+  auto usec =
+      duration_cast<microseconds>(now.time_since_epoch()).count() % 1000;
+
+  std::cout << "[TEST " << std::put_time(localtime, "%Y/%m/%d %H:%M:%S") << '.'
+            << std::setfill('0') << std::setw(6) << usec << std::setw(0)
+            << " coordinator:pid(" << boost::this_process::get_id() << ")] "
+            << s << " at line: " << lineno << std::endl
+            << std::flush;
 }
 
 // log a message and print the worker id as well.. used by fw_helper with no
 // worker id.
 void log(std::string s, int lineno, const char * /*filename*/, int /*id*/) {
-  char buf[128] = {0};
-  dunit::getTimeStr(buf, sizeof(buf));
-
-  fprintf(stdout, "[TEST 0:pid(%d)] %s at line: %d\n",
-          boost::this_process::get_id(), s.c_str(), lineno);
-  fflush(stdout);
+  using std::chrono::duration_cast;
```

Comment:
With the transition to boost::log for the test framework, what is still using this logger?

Reply:
Sadly I only changed tests/cpp/security code to log with Boost.Log
Thing is in theory this PR is supposed to be focused remove references for ACE time library.
Also, I tried to replace logging for Boost.Log on the old ITs, and the changes were huge.
So what do you think if a open a separate PR to adress this?

Reply:
No. If it's big change it isn't worth it. It was only worth it if it made refactoring this crusty log code easier.